rudder-web/src/main/resources/rudder-apache-common.conf

```diff
@@ -4,6 +4,11 @@
   Header add X-Chrome-Exponential-Throttling "disable"
 </LocationMatch>
 
+# Force using latest mode in IE.
+<LocationMatch "^/(?!rudder/api)">
```

Comment:
I still think this should match the location above.
By changing one or the other